packages/fast-components-react-base/src/radio/radio.spec.tsx

```diff
@@ -30,6 +30,12 @@ describe("radio", (): void => {
         expect((Radio as any).name).toBe(Radio.displayName);
     });
 
+    test("should have correct input radio type", () => {
```

Comment:
Should we test for the root as well here, which is div?

Reply:
Same comment as before, we should not test for things not related to accessibility. The component will not be any functionally different if we decide one day to make it a span etc.

Reply:
same comments as before as well :) but resolving to solve further down the road